### ISSUE TITLE
ADTPro will communicate but not function.

### ISSUE THREAD

**Issue by blainfaulkner** [ADTPro/adtpro#120]
Back Story,

I am relatively new to the ADTPro scene.  I have a background in IT but not in development.

The situation.

I have tried on several Mac computers to get ADTPro to work with no luck.  I have a provided ADTPro 2.0.3 disk purchased at the time of buying the supplies for this project.  The computer in question is an Apple //c with ROM version 3.  
The cables in use were purchased from apple2online.com.  I am using the null modem cable they offer with an FTDI-FT232RL usb to rs232 adapter. 

The long and the short is this; on every version of macOS (10.6-11.0.1) that I have tried I get the following symptoms.

Launch ADTPro 2.0.3 without issue using the ADTPro.command shell script.
Connect with the serial option with all default setting selected. (I have messed with them in an attempt with no change, thus I reverted back)
ADTPro Server side shows my FTDI adapter as the connected port, No activity above the progress bar, and connected below the progress bar
Then on the //c side of the world with serial selected at first boot I attempt to view the directory.
The //c reports Waiting for the host to reply, followed by host timeout.
On the server side of the world when I do this I see in the GUI "Request: directory contents" and below the status bar "Image Transfer aborted" is shown momentarily followed by just the path to the directory.
In my Java terminal windows I see no activity.

The fact that this is consistent across multiple OS's indicates user error but I just can't figure out what I am doing wrong even after reviewing the documentation.
I have provided images below incase my descriptions above are not clear.

<img width="321" alt="1" src="https://user-images.githubusercontent.com/75079795/100357378-9a128d00-2fc2-11eb-864a-f2c96fff88f0.png">
<img width="304" alt="2" src="https://user-images.githubusercontent.com/75079795/100357406-a565b880-2fc2-11eb-972b-12cba23a3a96.png">
<img width="313" alt="3" src="https://user-images.githubusercontent.com/75079795/100357412-a7c81280-2fc2-11eb-9bc2-c488e7b0a21d.png">
<img width="308" alt="4" src="https://user-images.githubusercontent.com/75079795/100357424-abf43000-2fc2-11eb-9e40-2275f77797c7.png">
<img width="532" alt="Terminal" src="https://user-images.githubusercontent.com/75079795/100357433-ae568a00-2fc2-11eb-9259-e8e142419f15.png">

Thanks so much in advance for any help you can provide.

**Comment by david-schmidt** [comment]
This is almost certainly a problem with your IIc serial cable.  One thing to try - on the server, click on File->Serial Configuration, and on the dialog box that comes up, tick the "Apple IIc w/Imagewriter Cable" and see if that changes anything.

**Comment by blainfaulkner** [comment]
Thanks you so much for looking at my post I greatly appreciate it.

I attempted the  "Apple IIc w/Imagewriter Cable" before and just failed to mention it. 

I will add however in the Java terminal window I was able to get some information to show up that might be relevant.  Take a look at the screen grab and let me know what you think if you don't mind.

This happened with all default settings and on both my macOS 10.6 machine and my macOS 11 machine.  I had gotten this before but it had stopped for some reason and figured it was due to me messing around with the settings but it is consistent at this point with the defaults. I have also attached the trace log as well if that might give more detail.

I am open to the possibility that it is the cable I just want to make sure that is the issue before I order a new one.

Thanks again.
<img width="561" alt="Screen Shot 2020-11-26 at 2 14 54 PM" src="https://user-images.githubusercontent.com/75079795/100386090-6f402d00-2ff2-11eb-9384-96b708132ba7.png">
[ADTProTrace.txt](https://github.com/ADTPro/adtpro/files/5604675/ADTProTrace.txt)

**Comment by david-schmidt** [comment]
That's consistent with wonky handshaking lines - we just aren't getting good 2-way communications.  It's also possible that there's a hardware fault in the IIc, but I find dodgy wiring to be far more common.  When you hear hoof beats, think horses, not zebras and all that.

**Comment by blainfaulkner** [comment]
Hey guys,

Sorry for the radio silence, I had to order parts.

So David hit the nail right on the head.  Turns out the cable I had ordered from apple2online.com was the issue.  I don't want to sling mud but my experiance with those guys was not good right down to what they ended up providing me.

To solve my issue I just ordered a fresh cable from Retro Floppy, smoothing sailing from that point forward.  I literally just pulled it out of the package and I was office to races.

Thanks again for the quick responses and hopefully this will help someone else that has this issue down the line.
